## 84% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 4
Expected to fail tests: 4
Passed tests: 43
```
```
IDs of failed tests: 727230,704613,727229,702775 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Failed tests (4)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feature/Allow_non_admin_to_add_members_to_group_chats_%2320396&PR_ID=20428&APK_NAME=20428.apk&TR_CASE_IDS=727230,704613,727229,702775">Rerun failed tests</a></li><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584402">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:120: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:41: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584401">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:111: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:100: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:87: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:81: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584399">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `accessibility id`: `community-title` </div><div>Device 1: `Text` is `Closed community` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the url https://status.app/c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56e159c16ce54f608e27d09ac5587770?auth=bbe9c2b8083452b3024e590ee1bde76b#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584360">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the deep link status.app://c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56e159c16ce54f608e27d09ac5587770?auth=bbe9c2b8083452b3024e590ee1bde76b#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584385">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join
E    Receiver was not navigated to the cats channel 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e81a662a50a24bb5a50cc8ba586311cd?auth=d09e93eb2c8e2ce14ea224c9d0318cf6#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc4d69164aea49c49fadcf8351a9d2f6?auth=c1e30d47a5536506d010649abe42da4d#292">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584398">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e81a662a50a24bb5a50cc8ba586311cd?auth=d09e93eb2c8e2ce14ea224c9d0318cf6#503">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc4d69164aea49c49fadcf8351a9d2f6?auth=c1e30d47a5536506d010649abe42da4d#478">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584404">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:214: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2430796775164c6190a83548a66e7846?auth=7ec537da4eb9b7761eaaefbf93598ae3#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584395">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#480">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#274">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (43)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584368">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e81a662a50a24bb5a50cc8ba586311cd?auth=d09e93eb2c8e2ce14ea224c9d0318cf6#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc4d69164aea49c49fadcf8351a9d2f6?auth=c1e30d47a5536506d010649abe42da4d#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584365">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e81a662a50a24bb5a50cc8ba586311cd?auth=d09e93eb2c8e2ce14ea224c9d0318cf6#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc4d69164aea49c49fadcf8351a9d2f6?auth=c1e30d47a5536506d010649abe42da4d#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584376">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e81a662a50a24bb5a50cc8ba586311cd?auth=d09e93eb2c8e2ce14ea224c9d0318cf6#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cc4d69164aea49c49fadcf8351a9d2f6?auth=c1e30d47a5536506d010649abe42da4d#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584356">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#360">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#231">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584392">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#287">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#169">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584391">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#107">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584366">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#161">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#79">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584367">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9723f608197344aba189db2a493fe0f5?auth=5b14f162a9a925d3a5c6d2c92c9f23fa#327">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e626475dcf0b4cfd9613242aee37c50c?auth=25f097dca987972552a7705edf8c9054#204">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b60df2dd21d34dc7b56ed5db66936d5d?auth=f1f221e0925984453db4619ce35a906e#202">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584384">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3db030e7a8b448faa42e3ac594cd0f31?auth=1401280da8ca0e82206ca414554823c8#133">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/81f8c18f46f94176946bb7e76d91f46e?auth=3dd88e0f1ced65397724c0e5f22b46e1#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584390">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#242">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584381">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#267">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#226">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584372">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#315">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#282">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584388">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#561">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#462">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584383">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#434">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#390">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584400">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#596">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#488">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584371">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#205">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#136">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584370">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#163">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#107">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584375">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#339">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#299">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584374">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#181">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584373">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e5bc8a3b80b24577ab809c3d85f3ad63?auth=8d2e2dbcf90936c5a994d2809d942ed2#404">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/996325074a9d4c45b71f82db2e1c6a79?auth=e27af2ca908321a10a462fcebe652c2a#360">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584362">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#144">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#120">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584357">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#543">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#397">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584369">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#402">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#260">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584359">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#344">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584380">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#451">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584394">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#482">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#337">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584355">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#206">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584354">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5f8a8c78bb694d46901fa89eb68e1150?auth=ab7b30c61373f194e826f2f0e017c49e#92">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/785bed4c37f5406e86e8a4305c7ebc8d?auth=4fe2ce42ea996d655ffd16723d1d3b92#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584361">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9b7bc9fc0d4d8b89ebcd82b88f6c25?auth=042c13d917554c422cf3c83701bb9191#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cea9932d1ee4196934a10217070a75d?auth=135c46bb83ad29e4fcd5adc862f9540f#212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584379">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9b7bc9fc0d4d8b89ebcd82b88f6c25?auth=042c13d917554c422cf3c83701bb9191#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cea9932d1ee4196934a10217070a75d?auth=135c46bb83ad29e4fcd5adc862f9540f#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584378">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a9b7bc9fc0d4d8b89ebcd82b88f6c25?auth=042c13d917554c422cf3c83701bb9191#57">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0cea9932d1ee4196934a10217070a75d?auth=135c46bb83ad29e4fcd5adc862f9540f#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584403">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2430796775164c6190a83548a66e7846?auth=7ec537da4eb9b7761eaaefbf93598ae3#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584389">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584358">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584397">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584382">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584377">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584393">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/88e0f11ddc7e4c5585e3a0c17a8a500f?auth=250101dff0d20c0f482caddc1af70563#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584386">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0bdc77cb49954d7b8b83f11956a131bc?auth=3edeb7f81fa0e571dde3425434d2af4f#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b48350bc81543a5b2711297adaed842?auth=ed65518dc540f8be7d98ceef58ab4a28#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584387">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0bdc77cb49954d7b8b83f11956a131bc?auth=3edeb7f81fa0e571dde3425434d2af4f#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b48350bc81543a5b2711297adaed842?auth=ed65518dc540f8be7d98ceef58ab4a28#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584364">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ddb533336d404b44bd2bdbd0100b8fb3?auth=788ec35dd83213efb0055ab4fa676aa1#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/95149cbd150c480da12f3303f25d6b26?auth=21e1c826aac83a1803c1bf76ecef7cae#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584363">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ddb533336d404b44bd2bdbd0100b8fb3?auth=788ec35dd83213efb0055ab4fa676aa1#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/95149cbd150c480da12f3303f25d6b26?auth=21e1c826aac83a1803c1bf76ecef7cae#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3584396">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ddb533336d404b44bd2bdbd0100b8fb3?auth=788ec35dd83213efb0055ab4fa676aa1#106">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/95149cbd150c480da12f3303f25d6b26?auth=21e1c826aac83a1803c1bf76ecef7cae#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/common/home/actions/view.cljs
+++ b/src/status_im/common/home/actions/view.cljs
@@ -365,20 +365,10 @@
           :sub-label           nil
           :chevron?            false}))
 
-(defn add-members-entry
-  [chat-id]
+(defn add-manage-members-entry
+  [chat-id admin?]
   (entry {:icon                :i/add-user
-          :label               (i18n/label :t/add-members)
-          :on-press            #(rf/dispatch [:open-modal :group-add-manage-members chat-id])
-          :danger?             false
-          :accessibility-label :add-members
-          :sub-label           nil
-          :chevron?            false}))
-
-(defn manage-members-entry
-  [chat-id]
-  (entry {:icon                :i/add-user
-          :label               (i18n/label :t/manage-members)
+          :label               (i18n/label (if admin? :t/manage-members :t/add-members))
           :on-press            #(rf/dispatch [:open-modal :group-add-manage-members chat-id])
           :danger?             false
           :accessibility-label :manage-members
@@ -433,8 +423,7 @@
         admin?          (get admins current-pub-key)]
     [(group-details-entry chat-id)
      (when inside-chat?
-       (when admin?
-         (manage-members-entry chat-id)))
+       (add-manage-members-entry chat-id admin?))
      (when (and admin? inside-chat?)
        (when config/show-not-implemented-features?
          (edit-group-entry)))

--- a/src/status_im/contexts/chat/group_details/view.cljs
+++ b/src/status_im/contexts/chat/group_details/view.cljs
@@ -27,9 +27,8 @@
       (rf/dispatch [:deselect-member public-key true]))))
 
 (defn add-member-contact-item-render
-  [{:keys [public-key] :as item} _ _ {:keys [group admin?]}]
-  (let [current-pk         (rf/sub [:multiaccount/public-key])
-        {:keys [contacts]} group
+  [{:keys [public-key] :as item} _ _ {:keys [group admin? current-pk]}]
+  (let [{:keys [contacts]} group
         member?            (contains? contacts public-key)
         checked?           (reagent/atom member?)]
     (if (or (= current-pk public-key) (and (not admin?) member?))
@@ -52,12 +51,13 @@
 
 (defn add-manage-members
   [{:keys [scroll-enabled? on-scroll]}]
-  (let [selected-participants      (rf/sub [:group-chat/selected-participants])
+  (let [theme                      (quo.theme/use-theme)
+        selected-participants      (rf/sub [:group-chat/selected-participants])
         deselected-members         (rf/sub [:group-chat/deselected-members])
         chat-id                    (rf/sub [:get-screen-params :group-add-manage-members])
         {:keys [admins] :as group} (rf/sub [:chats/chat-by-id chat-id])
-        theme                      (quo.theme/use-theme)
-        admin?                     (get admins (rf/sub [:multiaccount/public-key]))]
+        current-pk                 (rf/sub [:multiaccount/public-key])
+        admin?                     (get admins current-pk)]
     [rn/view {:flex 1 :margin-top 20}
      [rn/touchable-opacity
       {:on-press            #(rf/dispatch [:navigate-back])
@@ -78,8 +78,9 @@
        :render-section-header-fn       contact-list/contacts-section-header
        :render-section-footer-fn       contact-list/contacts-section-footer
        :content-container-style        {:padding-bottom 20}
-       :render-data                    {:group  group
-                                        :admin? admin?}
+       :render-data                    {:group      group
+                                        :admin?     admin?
+                                        :current-pk current-pk}
        :render-fn                      add-member-contact-item-render}]
      [rn/view {:style (style/bottom-container 30 theme)}
       [quo/button

--- a/src/status_im/contexts/chat/group_details/view.cljs
+++ b/src/status_im/contexts/chat/group_details/view.cljs
@@ -27,29 +27,37 @@
       (rf/dispatch [:deselect-member public-key true]))))
 
 (defn add-member-contact-item-render
-  [{:keys [public-key] :as item} _ _ {:keys [group]}]
+  [{:keys [public-key] :as item} _ _ {:keys [group admin?]}]
   (let [current-pk         (rf/sub [:multiaccount/public-key])
         {:keys [contacts]} group
         member?            (contains? contacts public-key)
         checked?           (reagent/atom member?)]
-    (fn []
-      (let [on-toggle #(group-chat-member-toggle member? (swap! checked? not) public-key)]
+    (if (or (= current-pk public-key) (and (not admin?) member?))
+      (fn []
         [contact-list-item/contact-list-item
-         (when (not= current-pk public-key)
+         {:disabled? true
+          :accessory {:disabled? true
+                      :type      :checkbox
+                      :checked?  @checked?}}
+         item])
+      (fn []
+        (let [on-toggle #(group-chat-member-toggle member? (swap! checked? not) public-key)]
+          [contact-list-item/contact-list-item
            {:on-press                on-toggle
             :allow-multiple-presses? true
             :accessory               {:type     :checkbox
                                       :checked? @checked?
-                                      :on-check on-toggle}})
-         item]))))
+                                      :on-check on-toggle}}
+           item])))))
 
 (defn add-manage-members
   [{:keys [scroll-enabled? on-scroll]}]
-  (let [selected-participants (rf/sub [:group-chat/selected-participants])
-        deselected-members    (rf/sub [:group-chat/deselected-members])
-        chat-id               (rf/sub [:get-screen-params :group-add-manage-members])
-        group                 (rf/sub [:chats/chat-by-id chat-id])
-        theme                 (quo.theme/use-theme)]
+  (let [selected-participants      (rf/sub [:group-chat/selected-participants])
+        deselected-members         (rf/sub [:group-chat/deselected-members])
+        chat-id                    (rf/sub [:get-screen-params :group-add-manage-members])
+        {:keys [admins] :as group} (rf/sub [:chats/chat-by-id chat-id])
+        theme                      (quo.theme/use-theme)
+        admin?                     (get admins (rf/sub [:multiaccount/public-key]))]
     [rn/view {:flex 1 :margin-top 20}
      [rn/touchable-opacity
       {:on-press            #(rf/dispatch [:navigate-back])
@@ -60,7 +68,7 @@
       {:size   :heading-1
        :weight :semi-bold
        :style  {:margin-left 20}}
-      (i18n/label :t/manage-members)]
+      (i18n/label (if admin? :t/manage-members :t/add-members))]
      [gesture/section-list
       {:key-fn                         :title
        :scroll-enabled                 @scroll-enabled?
@@ -70,7 +78,8 @@
        :render-section-header-fn       contact-list/contacts-section-header
        :render-section-footer-fn       contact-list/contacts-section-footer
        :content-container-style        {:padding-bottom 20}
-       :render-data                    {:group group}
+       :render-data                    {:group  group
+                                        :admin? admin?}
        :render-fn                      add-member-contact-item-render}]
      [rn/view {:style (style/bottom-container 30 theme)}
       [quo/button
@@ -159,17 +168,16 @@
                           :on-press            #(rf/dispatch [:chat.ui/mute chat-id (not muted)
                                                               (when-not muted
                                                                 constants/mute-till-unmuted)])}
-                         (when admin?
-                           {:accessibility-label :manage-members
-                            :customization-color color
-                            :icon                :i/add-user
-                            :label               (i18n/label :t/manage-members)
-                            :counter-value       (count contacts)
-                            :on-press            (fn []
-                                                   (rf/dispatch [:group/clear-added-participants])
-                                                   (rf/dispatch [:group/clear-removed-members])
-                                                   (rf/dispatch [:open-modal :group-add-manage-members
-                                                                 chat-id]))})]}]
+                         {:accessibility-label :manage-members
+                          :customization-color color
+                          :icon                :i/add-user
+                          :label               (i18n/label (if admin? :t/manage-members :t/add-members))
+                          :counter-value       (count contacts)
+                          :on-press            (fn []
+                                                 (rf/dispatch [:group/clear-added-participants])
+                                                 (rf/dispatch [:group/clear-removed-members])
+                                                 (rf/dispatch [:open-modal :group-add-manage-members
+                                                               chat-id]))}]}]
      [rn/section-list
       {:key-fn                         :title
        :sticky-section-headers-enabled false

